openshift/ci-operator/build-image/cr.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "ClusterLogging"
+metadata:
+  name: "example"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 1
+      storage: {}
+      redundancyPolicy: "ZeroRedundancy"
+      resources:
+        limits:
```

Comment:
probably a copy and pace but wouldnt we rather this be 'request' only or peg request and limit to a known value

Reply:
I don't know, but it works like this - what should it look like?

Reply:
in 3.11 we pegged the request and limit to the same value.  It has something to do with scheduling and ensuring the pod got all the memory it was configuring in the run script.  something like:
```
resources:
  limits:
    memory: 16G
  requests:
    memory: 16G
```